### PR TITLE
SONAR-10285 WS org/enable_support is now only available on SonarCloud

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: tests/src/test/java/org/sonarqube/pageobjects/Navigation.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public static Navigation create(Orchestrator orchestrator) {
     clearStorage(d -> d.getLocalStorage().clear());
     clearStorage(d -> d.getSessionStorage().clear());
     clearStorage(d -> clearBrowserLocalStorage());
-    return Selenide.open("/", Navigation.class);
+    return Selenide.open("/projects", Navigation.class);
```
Comment (simonbrandhof) [inline tests/src/test/java/org/sonarqube/pageobjects/Navigation.java]: why this change?

**Comment by simonbrandhof** [comment]
I suggest to do the same with all web services api/organizations. WDYT?

**Comment by julienlancelot** [comment]
It's not possible to disable all org ws as api/organization/search is used by front end, so we can disable all except this last one

**Comment by julienlancelot** [comment]
As discussed all org ws except search will be removed
